mlflow/pyfunc/__init__.py

```diff
@@ -228,7 +273,28 @@ def predict(*args):
         columns = [str(i) for i, _ in enumerate(args)]
         pdf = pandas.DataFrame(schema, columns=columns)
         result = model.predict(pdf)
-        return pandas.Series(result)
+        if not isinstance(result, pandas.DataFrame):
+            result = pandas.DataFrame(data=result)
+
+        if isinstance(elem_type, IntegralType):
+            result = result.select_dtypes(include=np.int)
```

Comment:
In Python 2, this omits any columns of datatype `np.long`. Can we make this `result.select_dtypes(include=[np.int, np.long])`?